test/autorest/generated/arraygroup/array.go

```diff
@@ -156,30 +156,41 @@ type ArrayOperations interface {
 	PutUUIDValid(ctx context.Context, arrayBody []string) (*http.Response, error)
 }
 
-// arrayOperations implements the ArrayOperations interface.
-type arrayOperations struct {
+// ArrayClient implements the ArrayOperations interface.
```

Comment:
So we are going to output the interface definition for all management plane but not data-plane?

Reply:
Correct, that will happen in a separate PR.